### ISSUE TITLE
win 7 error [Bug]

### ISSUE THREAD

**Issue by ruboard** [henrypp/simplewall#1787]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

3.8.2

### Windows version

6.1 build 7601

### Steps to reproduce

launching the program. default settings.

### Actual behavior

A window pops up: something went wrong. see log.

package log is empty!
everything else seems to be working!

### Logs

simplewall_debug.log :
"Error","09.05.2024 19:30:50","FwpmFilterAdd0","0x80320001","FWPM_LAYER_ALE_AUTH_CONNECT_V4\Internal\BlockConnection","3.8.2","6.1 build 7601"
"Error","09.05.2024 19:30:50","FwpmFilterAdd0","0x80320001","FWPM_LAYER_ALE_AUTH_CONNECT_V6\Internal\BlockConnection","3.8.2","6.1 build 7601"

**Comment by henrypp** [comment]
так, вобщем баг исправил, но это совсем не влияет на работу програмы, просто не до конца на 7 протестировал и не все удаленое вернул, вот исправленый бинарь (x64):

[simplewall.zip](https://github.com/henrypp/simplewall/files/15266153/simplewall.zip)

**Comment by blzos** [comment]
Windows 7 снова в деле! Обновил свою 3.6.7

**Comment by ruboard** [comment]
> так, вобщем баг исправил, но это совсем не влияет на работу програмы, просто не до конца на 7 протестировал и не все удаленое вернул, вот исправленый бинарь (x64):
> 
> [simplewall.zip](https://github.com/henrypp/simplewall/files/15266153/simplewall.zip)

да. так точно. просто не выскакивает окошко с ошибкой.
лога по прежнему нет. и сабж не реагирует на появление новых программ.

**Comment by blzos** [comment]
> сабж не реагирует на появление новых программ.

И правда не появляется окошко, что программа ломится в интернет, он её просто не пускает

**Comment by henrypp** [comment]
опять говнокод который исправил[simplewall.zip](https://github.com/henrypp/simplewall/files/15268298/simplewall.zip), но оно и раньше не выскакивало, а просто в лог писало...

**Comment by blzos** [comment]
> перекачай релиз

Заработало как надо, будем наблюдать дальше...

**Comment by Komprogue** [comment]
Есть баг с ярлыком в трее: после запуска приложение не отображается ни там, ни на панели задач (как и при его сворачивании), хотя работает в фоне и разворачивается по двойному клику на ярлык.

**Comment by henrypp** [comment]
@Komprogue ни разу не понятно о чём ты вообще и как это относится к теме

**Comment by blzos** [comment]
> Есть баг с ярлыком в трее: 

Возможно программа раньше запускалась из другой папки, очистите куш иконок трея:

https://github.com/henrypp/simplewall/issues/1695
или
https://otvet.mail.ru/question/39914075?ysclid=lw713ac95h81825566
